e2e/cypress/integration/emoji/sorted_emojis_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('M16739 - Filtered emojis are sorted', () => {
+    before(() => {
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.clearLocalStorage(/recent_emojis/);
+    });
+
+    it('By recency', async () => {
```

Comment:
The async here causes the tests to run and passed immediately.
```suggestion
    it('By recency', () => {
```

Reply:
you're right, done